app/models/job.ts

```diff
@@ -403,10 +506,10 @@ export class Job extends Record implements JobRecord {
   /**
    *  Checks the status of the job to see if the job is paused.
    *
-   * @returns true if the `Job` is paused
+   * @returns true if the `Job` is paused or previewing
    */
   isPaused(): boolean {
-    return this.status === JobStatus.PAUSED;
+    return [JobStatus.PAUSED, JobStatus.PREVIEWING].includes(this.status);
```

Comment:
I'm not sure if this should be more strict and only be true if the status is PAUSED, but it seemed like it should be considered paused if it were not completed and not running. In any case this function is only used on a test, so maybe not important.